timetracker/entries/views.py

```diff
@@ -1,3 +1,33 @@
 from django.shortcuts import render
+from django.http import HttpResponse
+from entries.models import Client, Project, Entry
 
 # Create your views here.
+
+
+def clients(request):
+
+    clients = Client.objects.all()
+    return render(request, 'template.html', {'category': 'Clients', 'object_list': clients})
+
+
+def projects(request):
+
+    projects = Project.objects.all()
+    return render(request, 'template.html', {'category': 'Projects', 'object_list': projects})
+
+
+def getEntries(client):
+
+    summary = Entry.objects.order_by('project').filter(project__client__name=client)
+    return summary
+
+
+def clientsummary(request):
+
+    clients = Client.objects.all()
+    results = {}
```

Comment:
Yes I observed that they are not sorted.
so do you mean from:
{ "key1" : [value1, value2, value3], "key2" : [value1, value2] }
 to:
[ [key1, [value1, value2, value3]] , [key2, [value1, value2]] ]